pkg/apis/servicecatalog/types.go

```diff
@@ -60,8 +60,31 @@ type ServiceBrokerSpec struct {
 	// CABundle is a PEM encoded CA bundle which will be used to validate a Broker's serving certificate.
 	// +optional
 	CABundle []byte
+
+	// RelistBehavior specifies the type of relist behavior the catalog should exhibit
+	// when relisting ServiceClasses available from a broker
+	RelistBehavior ServiceBrokerRelistBehavior
+
+	// RelistDuration is the frequency by which a controller will relist the broker
+	// when the RelistBehavior is set to ServiceBrokerRelistBehaviorDuration
+	RelistDuration *metav1.Duration
+
+	// RelistRequests is a strictly increasing integer counter that can be manually incremented
```

Comment:
If we are claiming that this is a strictly increasing integer, then we should add a check in the ValidateUpdate function of the brokerRESTStrategy that validates that the new value is not less than the old value.